WSL/wsl2-install.md

```diff
@@ -11,22 +11,14 @@ ms.localizationpriority: high
 
 # Installation Instructions for WSL 2

```

Comment:
Moving wsl2-install.md to WSL/wsl2-install.md will break the existing links we have set. 
@mattwojo  is there a way we can set up a redirect so we don't break people trying to access wsl2-install.md?